From 5d5ba8a2b8010db6bee68bd712f829cb737c9ac1 Mon Sep 17 00:00:00 2001
From: Lei Maohui <leimaohui@fujitsu.com>
Date: Fri, 10 Mar 2023 03:48:46 +0000
Subject: [PATCH] dhcpcd.8: Fix conflict error when enable multilib.

Error: Transaction test error:
 file /usr/share/man/man8/dhcpcd.8 conflicts between attempted
 installs of dhcpcd-doc-9.4.1-r0.cortexa57 and
 lib32-dhcpcd-doc-9.4.1-r0.armv7ahf_neon

The differences between the two files are as follows:
@@ -821,7 +821,7 @@
 If you always use the same options, put them here.
 .It Pa /usr/libexec/dhcpcd-run-hooks
 Bourne shell script that is run to configure or de-configure an interface.
-.It Pa /usr/lib64/dhcpcd/dev
+.It Pa /usr/lib/dhcpcd/dev
 Linux
 .Pa /dev
 management modules.

It is just a man file, there is no necessary to manage multiple
versions.

Upstream-Status: Inappropriate [oe specific]
Signed-off-by: Lei Maohui <leimaohui@fujitsu.com>

---
 src/dhcpcd.8.in | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/dhcpcd.8.in b/src/dhcpcd.8.in
index 93232840..09930a31 100644
--- a/src/dhcpcd.8.in
+++ b/src/dhcpcd.8.in
@@ -824,7 +824,7 @@ Configuration file for dhcpcd.
 If you always use the same options, put them here.
 .It Pa @SCRIPT@
 Bourne shell script that is run to configure or de-configure an interface.
-.It Pa @LIBDIR@/dhcpcd/dev
+.It Pa /usr/<libdir>/dhcpcd/dev
 Linux
 .Pa /dev
 management modules.
